client/my-sites/plans/jetpack-plans/product-lightbox/product-details.tsx

```diff
@@ -0,0 +1,49 @@
+import { useMobileBreakpoint } from '@automattic/viewport-react';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import FoldableCard from 'calypso/components/foldable-card';
+import { SelectorProduct } from '../types';
+import DescriptionList from './description-list';
+
+type ProductDetailsProps = {
+	product: SelectorProduct;
+};
+
+const ProductDetails: React.FC< ProductDetailsProps > = ( { product } ) => {
+	const isMobile = useMobileBreakpoint();
+	const translate = useTranslate();
+	const [ expandedDetailType, setExpandedDetailType ] = useState( '' );
+
+	const productDetails = [
+		{ type: 'includes', title: translate( 'Includes' ), items: product.whatIsIncluded },
+		{ type: 'benefits', title: translate( 'Benefits' ), items: product.benefits },
+	];
+
+	return (
+		<>
+			{ productDetails.map( ( { type, title, items }, index, infoList ) => (
+				<div className="product-lightbox__detail-list" key={ type }>
+					{ isMobile ? (
+						<FoldableCard
+							hideSummary
+							header={ title }
+							clickableHeader={ true }
+							expanded={ expandedDetailType === type }
+							onOpen={ () => setExpandedDetailType( type ) }
```

Comment:
I think we can remove these two props because users may want to expand both sections at the same time.

Reply:
I guess in the planning meeting with the design team we decided to go with this approach, I might be wrong though. I believe it was discussed to have only one open at a time so that users unnecessarily don't need to scroll

Reply:
The prototype and Figma show them both openable at the same time. Let us go with that for now and come back later if we see scrolling issue.